### ISSUE TITLE
Confusing inconsistency in alembic migration for add_track_principal

### ISSUE THREAD

**Issue by marengeti** [indico/indico#4332]
The file:
https://github.com/indico/indico/blob/master/indico/migrations/versions/20191111_1820_2496c4adc7e9_add_track_principal.py

Inconsistency:
```
Revises: eefba82b42c5
down_revision = '4e459d27adab'
```

**Comment by ThiefMaster** [comment]
I guess it's only the comment that's wrong?

**Comment by marengeti** [comment]
I think so. But better to correct the comment that cause more time wasting for anyone using this code and encountering that inconsistency and being forced to figure it out.

**Comment by ThiefMaster** [comment]
sure, i just wanted to confirm that it's the comment which is wrong and not the actual reference. wanna send a PR to fix it in master?

**Comment by marengeti** [comment]
The revid in the docstring `Revises: eefba82b42c5` seems to have been moved to earlier in the chain (Add track_groups table). I am assuming the following dn history sequence is the correct one for core db?

```
$ indico db history
4e459d27adab -> 2496c4adc7e9 (head), Add track principal
39a25a873063 -> 4e459d27adab, Add tables for editing revisions
bb522e9f9066 -> 39a25a873063, Add editing file_types table
a2472148d2c5 -> bb522e9f9066, Add files table
f6fba869a27c -> a2472148d2c5, Add signature token column to User
0f20a5e4018e -> f6fba869a27c, Add uuid column to event_abstracts.abstracts
1b741c9123f6 -> 0f20a5e4018e, Add new abstract state
eefba82b42c5 -> 1b741c9123f6, Add columns for program codes
620b312814f3 -> eefba82b42c5, Add track_groups table
06a4ec717b84 -> 620b312814f3, Migrate indico-weeks-view theme name
7024f7f66e20 -> 06a4ec717b84, Switch from JSON to JSONB
a83e77e11e36 -> 7024f7f66e20, Remove assistance/vc columns from rb
ad0625914645 -> a83e77e11e36, Add Location.is_deleted
8521bce91242 -> ad0625914645, Room.is_active -> is_deleted
fe73a07da0b4 -> 8521bce91242, Remove location is_default
3e4a0c08eae6 -> fe73a07da0b4, Add Room.notification_emails column
081d4c97060a -> 3e4a0c08eae6, Fix truncated constraint names (sqlalchemy 1.3)
7aabedfb5e3a -> 081d4c97060a, Add end notifications
416f9c877300 -> 7aabedfb5e3a, Make some room cols not nullable
ff49d8d05ce7 -> 416f9c877300, Add reservation link table
7ec3949a21c7 -> ff49d8d05ce7, Disallow empty rejection reasons
579a36843848 -> 7ec3949a21c7, Use enum for resv occurrence state
cbe630695800 -> 579a36843848, Use enum for reservation state
252c0015c9a0 -> cbe630695800, Add room principals table
4f98f2f979c7 -> 252c0015c9a0, Add missing blocking principal constraints
7e03b2262e9e -> 4f98f2f979c7, Remove RoomAttribute.is_required
868c04697dd7 -> 7e03b2262e9e, Add room features
db32adb8fc4e -> 868c04697dd7, Remove reservation-equipment association
7749141f3e08 -> db32adb8fc4e, Remove equipment nesting
ec410be271df -> 7749141f3e08, Remove room attribute nesting
27c45c384d65 -> ec410be271df, Make room attributes global
0117bd0fa784 -> 27c45c384d65, Make equipment types global
c5c21246445a -> 0117bd0fa784, Store room coordinates as numbers
7a72d63acba9 -> c5c21246445a, Rename aspects to map areas
c0888cb57c58 -> 7a72d63acba9, Update map aspects structure
f7ab9ee32bdf -> c0888cb57c58, Remove holidays table
55893713f6b7 -> f7ab9ee32bdf, Add verbose_name column
93985a8c11ed -> 55893713f6b7, Add room_name_format
a13e25814c4c -> 93985a8c11ed, Remove room thumbnail
66ecbb1c0ddd -> a13e25814c4c, Add favorite rooms table
813ea74ce8dc -> 66ecbb1c0ddd, Add natsort function
c820455976ba -> 813ea74ce8dc, Add AttachmentFolder.is_hidden
093533d27a96 -> c820455976ba, Add include_description to reminders
9c4418d7a6aa -> 093533d27a96, Add visibility options to contribution fields
2af245be72a6 -> 9c4418d7a6aa, Add session types table
566d5de4e0e5 -> 2af245be72a6, Add columns to review questions and review ratings tables.
1d512a9ebb30 -> 566d5de4e0e5, Add is_private column to contribution types table
790a06790309 -> 1d512a9ebb30, Add default session to tracks
640584a3987e -> 790a06790309, Rename roles to permissions
f1eee7b4880a -> 640584a3987e, Add event roles to acls
<base> -> f1eee7b4880a, Add event roles table
```

**Comment by ThiefMaster** [comment]
Yes, it should say "Revises: 4e459d27adab"
